zerver/lib/send_email.py

```diff
@@ -34,6 +36,17 @@ def tokenized_no_reply_address() -> str:
             return parseaddr(settings.TOKENIZED_NOREPLY_EMAIL_ADDRESS)[1].format(token=generate_key())
         return FromAddress.NOREPLY
 
+    @staticmethod
+    def from_name_of_security_emails(language: Optional[str]=None,
+                                     user_profile: Optional[UserProfile]=None) -> str:
+        if language:
+            with override_language(language):
+                return _("Zulip Account Security")
+        if user_profile is not None:
+            with override_language(user_profile.default_language):
+                return _("Zulip Account Security")
```

Comment:
We can avoid a bit of code duplication by doing 
```
if language is None:
    assert user_profile is not None
    language = user_profile.default_language
...
```

I would also call this function `security_email_from_name`.